src/views/DepositWithdraw/components/DepositInfo/index.tsx

```diff
@@ -106,7 +106,7 @@ export const DepositInfo: React.FC<IDepositInfo> = ({ farmSelected }) => {
             <SmallInfoCard
               iconType={'question'}
               statTitle={'Farm Ownership'}
-              statContent={`${ownershipShare.toFixed(2)}%`}
+              statContent={`${ownershipShare >= 0.01 || ownershipShare === 0 ? ownershipShare.toFixed(2) : '< 0.01'}%`}
```

Comment:
For the sake of consistency, I'd also refactor this into a little function to format this. This way, whenever we make future changes to the code, it's easier to modify. Also, could we get some unit tests for this?